otter/convergence/service.py

```diff
@@ -211,39 +217,60 @@ def _execute_steps(steps):
 
 
 @do
-def convergence_exec_data(tenant_id, group_id, now, get_all_convergence_data):
+def convergence_exec_data(
+        tenant_id, group_id, now,
+        get_all_launch_server_data=get_all_launch_server_data,
+        get_all_launch_stack_data=get_all_launch_stack_data):
     """
     Get data required while executing convergence
     """
     sg_eff = Effect(GetScalingGroupInfo(tenant_id=tenant_id,
                                         group_id=group_id))
-    gather_eff = get_all_convergence_data(tenant_id, group_id, now)
-    try:
-        data = yield parallel([sg_eff, gather_eff])
-    except FirstError as fe:
-        six.reraise(*fe.exc_info)
-    [(scaling_group, manifest), (servers, lb_nodes)] = data
+
+    (scaling_group, manifest) = yield sg_eff
+
+    # not sure how to handle this...
```

Comment:
It's safe to delete this now, since there's no more FirstError from parallel to unwrap.